### PR TITLE
Check for existence of 'apps' and 'users' in the configuration file.

### LINKED ISSUE #4: `apps` may not exist in permissions.yml
/usr/bin/sync-accounts:34:in `read_permissions': undefined method`each' for nil:NilClass (NoMethodError)
    from /usr/bin/sync-accounts:98:in `<main>'

### PR DESCRIPTION
**Pull request by waratuman**
Fixes #4

### CODE REVIEW

**Comment by waratuman** [comment]
@malomalo can you approve this?

File: sync-accounts [hunk 1]
```diff
@@ -28,11 +28,16 @@ def read_permissions(url)
   end
   
   # Add keys to users and apps
-  yaml['users'].each do |user|
-    permissions['users'][user] = github_keys(user)
+  if (yaml['users'])
```
Comment (malomalo) [inline sync-accounts]: Remove parens and just use `yaml['users']` or `yaml.has_key('users')`

File: sync-accounts [hunk 2]
```diff
@@ -28,11 +28,16 @@ def read_permissions(url)
...
+    yaml['users'].each do |user|
+      permissions['users'][user] = github_keys(user)
+    end
   end
-  yaml['apps'].each do |app, users|
-    permissions['apps'][app] = users.map{|u| github_keys(u) }.flatten
+
+  if (yaml['apps'])
```
Comment (malomalo) [inline sync-accounts]: Remove parens and just use `yaml['users']` or `yaml.has_key('users')`

**Review by malomalo — CHANGES_REQUESTED** [review]

**Comment by waratuman** [comment]
@malomalo review again.

**Pull request merged**